### ISSUE TITLE
``python-version`` seems to be missing from outputs when installing ``PyPy``

### ISSUE THREAD

**Issue by DanielNoord** [actions/setup-python#345]
**Description:**
[Run](https://github.com/PyCQA/pylint/runs/5376859270?check_suite_focus=true)
[Workflow](https://github.com/PyCQA/pylint/blob/main/.github/workflows/ci.yaml)

In the above job the `Restore Python Virtual Environment` step uses the following key:
```yaml
 ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
            needs.prepare-tests-pypy.outputs.python-key }}
```
See L479 of the linked workflow file.

However, if you look at the `key` shown on L15 of that step you see that the `python-version` is missing.

This step does work for the `cpython` jobs on both Linux and Windows, but not for `PyPy`.
Is this a known issue?

Edit: I checked and this behaviour did not occur a couple of weeks ago. Might this be related to #342?

**Action version:**
3.0.0

**Platform:**
- [x] Ubuntu

**Runner type:**
- [x] Hosted

**Tools version:**
`pypy-3.6`, `pypy-3.7` and possibly more.

**Repro steps:**  
See the run and workflow linked above.

**Expected behavior:**
The `setup-python` action to have `python-version` in its outputs when `PyPy` is installed.

**Actual behavior:**
`python-version` seems to be missing.

**Comment by marko-zivic-93** [comment]
Hello @DanielNoord 
Thank you for your report!
We will take a look at this issue and come back with results as soon as possible.

**Comment by dsame** [comment]
@marko-zivic-93 please review my PR

**Comment by DanielNoord** [comment]
I think this can be closed now that #365 has been merged. @brcrista Gentle ping since you merged that PR 😄
